.travis.yml

```diff
@@ -29,11 +29,11 @@ after_success:
     npm run coverage:upload;
   fi
   if [[ $TRAVIS_BRANCH == 'release' && $TRAVIS_PULL_REQUEST == 'false' ]]; then
```

Comment:
Not sure about the "release" branch. I changed this weeks ago but I think we reverted to using the "master".